src/metax_api/services/file_service.py

```diff
@@ -647,6 +629,38 @@ def get_directory_contents(cls, identifier=None, path=None, project_identifier=N
 
         return contents
 
+    @classmethod
+    def _get_cr_if_relevant(cls, cr_identifier, directory, request):
+        if cr_identifier:
```

Comment:
The if-block seem obsolete now that the code is extracted to a method.

Reply:
true that :)